envs/env.py

```diff
@@ -336,6 +346,19 @@ def _start_external_processes(self, reset_time=True):
             self.episode_start_time = time.time()
         logging.info("All commands executed. Episode started!")
 
+    def set_random_variation_step(self):
+        random_step = int(self.max_time_steps_per_episode/3) if self._is_testing else np.random.randint(self.max_time_steps_per_episode/4, 3*self.max_time_steps_per_episode/4)
+        return random_step
```

Comment:
As it's a class method you don't need to set that variable and then return it, you can just either directly return the expression with `return xyz if a else b` , or maybe directly set the class variable like `self.name_of_the_variable = ...`